README.md

```diff
@@ -0,0 +1,37 @@
+# Ed25519 Grant Handler
+This grant handler allows users to obtain OAuth tokens from WSO2 Identity by exchanging an SSH certificate of type ssh-ed25519-cert-v01@openssh.com.
+
+## Installing
+### Building from source
+1. Clone the repository with `git clone https://github.com/gayashanbc/x509-ed25519-grant-handler.git`.
+2. Build using Maven `mvn clean install`.
+
+You can find the `wso2is-identity-samples-oauth2-ssh-ed25519-cert-v01-grant-1.0.0.jar` inside the `target` folder of the project root directory.
+
+### Downloading the release artifact
+1. Download [wso2is-identity-samples-oauth2-ssh-ed25519-cert-v01-grant-1.0.0.jar](https://github.com/gayashanbc/x509-ed25519-grant-handler/releases/download/1.0.0/wso2is-identity-samples-oauth2-ssh-ed25519-cert-v01-grant-1.0.0.jar).
+
+## Configuring
+1. Coyy `wso2is-identity-samples-oauth2-ssh-ed25519-cert-v01-grant-1.0.0.jar` to `<IS_HOME>/repository/components/dropins` directory.
```

Comment:
```suggestion
1. Copy `wso2is-identity-samples-oauth2-ssh-ed25519-cert-v01-grant-1.0.0.jar` to `<IS_HOME>/repository/components/dropins` directory.
```

Reply:
Resolved in https://github.com/gayashanbc/x509-ed25519-grant-handler/pull/4/commits/0884db8a2eaffdf814eab1ec2f63687e735c8db8